pkg/cmd/admin/policy/remove_from_project.go

```diff
@@ -109,8 +121,24 @@ func (o *RemoveFromProjectOptions) Run() error {
 			if err != nil {
 				return err
 			}
+
+			if diff := existingGroups.Difference(currBinding.Groups); len(diff) != 0 {
+				fmt.Fprintf(o.Out(), "Removing %s/%s from groups %v in project %s.\n", currBinding.RoleRef.Namespace, currBinding.RoleRef.Name, diff.List(), o.BindingNamespace)
```

Comment:
for cluster roles: `Removing admin role from...`
for namespaced roles: `Removing ns/thinger role from...`